Entities/AtomGroup.cpp

```diff
@@ -1501,9 +1475,7 @@ Vector AtomGroup::PushTravel(Vector &position,
 
     bool hFlipped = m_pOwnerMO->m_HFlipped;
 
-    // Compute and scale the actual on-screen travel trajectory for this travel, using
-    // the passed in travel time and the pixels-per-meter constant.
-    Vector trajectory = velocity * travelTime * c_PPM;
+    Vector trajectory = velocity * travelTime * c_PPM; //!< Trajectory length in pixels.
```

Comment:
No need to do the doxygen style comments for inline ones outside of header definitions. We need the `!<` because otherwise generated docs won't pick up on it properly I think, but here you can just do `//`

Reply:
Does that apply in line 958 as well?

Reply:
Yep, you only need that formatting on member variables in header files.